database-plugins/src/test/java/io/cdap/plugin/db/connector/DBConnectorTest.java

```diff
@@ -136,10 +137,7 @@ private void testSample(DBConnector connector) throws IOException {
 
   private void testBrowse(DBConnector connector) throws IOException {
     // browse DB server
-    BrowseDetail detail = connector.browse(new MockConnectorContext(new MockConnectorConfigurer()),
-                                           BrowseRequest.builder("/").build());
-    Assert.assertTrue(detail.getTotalCount() > 0);
-    Assert.assertTrue(detail.getEntities().size() > 0);
+    BrowseDetail detail = null;
```

Comment:
I can't comment on the existing code below, but there are several System.out.println() statements that should be removed.

Reply:
removed.